src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -1273,16 +1273,17 @@
                                         <RowDefinition Height="Auto"/>
                                     </Grid.RowDefinitions>
                                     <StackPanel Margin="0,10,0,5" Grid.Row="0" Orientation="Horizontal">
-                                        <TextBlock  FontSize="12" Text="Library"></TextBlock>
+                                        <TextBlock  FontSize="12" VerticalAlignment="Center" Text="Library"></TextBlock>
```

Comment:
I did not notice this before, it seems the text `Library` is missing localization this way

Reply:
Thanks for noticing, added a resource for that.